packages/api/lambdas/bulk-operation.js

```diff
@@ -72,14 +72,16 @@ async function applyWorkflowToGranules({
  * @param {string} [payload.index] - Optional parameter of ES index to query.
  * Must exist if payload.query exists.
  * @param {Object} [payload.ids] - Optional list of granule IDs to bulk operate on
- * @param {Function} [unpublishGranuleFunc] - Optional function to delete the
- * granule from CMR. Useful for testing.
+ * @param {Function} [removeGranuleFromCmrFunction] - passed in function used for test mocking
  * @returns {Promise}
  */
 async function bulkGranuleDelete(
   payload,
-  unpublishGranuleFunc = unpublishGranule
+  removeGranuleFromCmrFunction
 ) {
+  const concurrency = payload.concurrency || 10;
+  process.env.dbMaxPool = payload.maxDbConnections || concurrency;
```

Comment:
`payload.maxDbConnections` needs to be added to the JSDocs.

Reply:
e82fd4364f